.eslintrc

```diff
@@ -24,6 +24,13 @@
     "no-ex-assign": ["error"],
     "no-constant-condition": ["off"],
     "no-return-await": ["error"],
+    "no-restricted-syntax": [
```

Comment:
I realise we could also make the error message a required arg for the assert function

```
export function assert<T>(
  predicate: T,
  errorMessage?: string,
): asserts predicate {
  if (!predicate) {
    throw new Error(errorMessage ?? 'Error');
  }
}
```

Reply:
Right, so why not make that change first?

Reply:
I arrived at this right at the end after realising assert was something we implemented

Reply:
I've made the change to assert